### PR TITLE
Add SelfKisser addresses

### PR DESCRIPTION
**Pull request by biancabuzea200**
Add SelfKisser addresses to Remix guide

### CODE REVIEW

File: docs/Developers/tutorials/Remix.md [hunk 1]
```diff
@@ -5,13 +5,38 @@ keywords: [Remix, read access]
 ---
 # Consuming Oracle Data (Remix)
```
Comment (teghnet) [inline docs/Developers/tutorials/Remix.md:6]:
```suggestion
# Consuming Oracle Data
```

File: docs/Developers/tutorials/Remix.md [hunk 2]
```diff
@@ -5,13 +5,38 @@ keywords: [Remix, read access]
...
+| Gnosis Mainnet                | [0x0Dcc19657007713483A5cA76e6A7bbe5f56EA37d](https://gnosisscan.io/address/0x0Dcc19657007713483A5cA76e6A7bbe5f56EA37d)                       |
+| Mantle Testnet                | [0x9ee0DC1f7cF1a5c083914e3de197Fd1F484E0578](https://explorer.sepolia.mantle.xyz/address/0x9ee0DC1f7cF1a5c083914e3de197Fd1F484E0578)         |
+| Scroll Sepolia                | [0x0Dcc19657007713483A5cA76e6A7bbe5f56EA37d](https://sepolia.scrollscan.com/address/0x0Dcc19657007713483A5cA76e6A7bbe5f56EA37d#code)         |
+| zkSync Sepolia                | [0x25f594edde4f58A14970b2ef6616badBa4B1CdDD](https://sepolia.explorer.zksync.io/address/0x25f594edde4f58A14970b2ef6616badBa4B1CdDD#contract) |
+| Optimism Sepolia              | [0xfF619a90cDa4020897808D74557ce3b648922C37](https://sepolia-optimism.etherscan.io/address/0xfF619a90cDa4020897808D74557ce3b648922C37#code)  |
+| Berachain Bartio              | [0x2FFCAfF4BcF6D425c424f303eff66954Aa3A27Fd](https://bartio.beratrail.io/address/0x2FFCAfF4BcF6D425c424f303eff66954Aa3A27Fd)                 |
+
+
 ### Consuming Oracle Data
```
Comment (teghnet) [inline docs/Developers/tutorials/Remix.md:33]:
```suggestion
### Consuming Oracle Data (Remix)
```

**Comment by teghnet** [comment]
minor detail @biancabuzea200

**Review by jar-o — DISMISSED** [review]
lgtm with one nit

File: docs/Developers/tutorials/Remix.md [hunk 3]
```diff
@@ -3,15 +3,40 @@ sidebar_position: 1
...
+
+:::
+
+| Testnet Network               | SelfKisser Address                                                                                                                           |
+| ----------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------- |
+| Ethereum Sepolia              | [0x0Dcc19657007713483A5cA76e6A7bbe5f56EA37d](https://sepolia.etherscan.io/address/0x0Dcc19657007713483A5cA76e6A7bbe5f56EA37d#code)           |
+| Base Sepolia                  | [0x70E58b7A1c884fFFE7dbce5249337603a28b8422](https://sepolia.basescan.org/address/0x70E58b7A1c884fFFE7dbce5249337603a28b8422#code)           |
+| Arbitrum Sepolia              | [0xc0fe3a070Bc98b4a45d735A52a1AFDd134E0283f](https://sepolia.arbiscan.io/address/0xc0fe3a070Bc98b4a45d735A52a1AFDd134E0283f#code)            |
+| Polygon zkEVM Testnet Cardona | [0xCce64A8127c051E784ba7D84af86B2e6F53d1a09](https://cardona-zkevm.polygonscan.com/address/0xCce64A8127c051E784ba7D84af86B2e6F53d1a09)       |
```
[the remark this answers is not in the document]
Comment (biancabuzea200) [reply docs/Developers/tutorials/Remix.md]: great catch, thank you. Fixed :)

**Pull request merged**